LathBotFront/Commands/ReactionCommands.cs

```diff
@@ -110,6 +118,10 @@ public async Task Pat(CommandContext ctx, [Description("Who to pat")] DiscordMem
                 await ctx.RespondAsync(new DiscordMessageBuilder().WithStickers([ctx.Guild.Stickers[1013888253272801310]]));
             else if (member.Id == 387325006176059394)//julian
                 await ctx.RespondAsync($"Julian!, you just got pat by {ctx.Member.Mention}");
+            else if (member.Id == 312936081819697154)//Ryguy6001
+                await ctx.RespondAsync($"Ryguy doesn't like getting patted.");
+            else if (member.Id == 671485545288826900)//Tophat
+                await ctx.RespondAsync($"Tophat does not enjoy this, please stop.");//Tophat also requested this for himself.
```

Comment:
too bad, everyone can be pat, please remove
he can yell at people who try individually